File: sbe/binance/spot_3_2/generated-python/spot_sbe/OrderCapacity.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple
from io import BytesIO

class OrderCapacity:
    """Enum values."""
    PRINCIPAL = 1
    AGENCY = 2
    NON_REPRESENTABLE = 254
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1

File: sbe/binance/spot_3_1/generated-php/spot_sbe/OrderCapacity.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class OrderCapacity
{
    public const PRINCIPAL = 1;
    public const AGENCY = 2;
    public const NONREPRESENTABLE = 254;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}
